docs/tenancy/1.x/tutorial-non-tenant.md

```diff
@@ -0,0 +1,161 @@
+    
+    ...
+  ];
+```
+
+## Status
+
+At this point you application is configured to load any tenant routes when a tenant is identified. When a tenant is not identified, the routes located in `routes/web.php` will be used instead.
+
+## Tips and Reminders
+### Namespaceing
+As with any Laravel application; if you are storing controllers in a different location than in `app\Http\Controllers` you will need to use the namespace option within your routes.
+
+`routes\tenant.php`
+```php
+Route::namespace('Tenant')->group(function () {
+    // Controllers Within The "App\Http\Controllers\Tenant" Namespace
+});
+```
+
+### Sub-site by domains
+When using subdomains to identify your non-tenant sub-sites you will need to use the route option within your routes file.
+`routes\web.php`
+```php
+Route::domain('admin.myapp.com')->group(function () {
+    Route::get('/', function () {
+        // This is the root route for the admin site
+    });
+});
+
+Route::domain('www.myapp.com')->group(function () {
+    Route::get('/', function () {
+        // This is the root route for the public site
+    });
+});
+
+Route::get('/shared', function() {
+  //This is a route that is shared with both the admin site and the public site
+})
+```
+
+### Accessing `OnTenant` models from a "sub-site"
+Before a model that uses the `OnTenant` trait can be used a tenant needs to be identified. Because of this it is best practice to avoid using tenant models on a non-tenant site.
+
+In events where this access is still needed, a tenant will first need to be identified within your code.
+
+The following example assumes you are trying to create an admin view that lists all products a specific tenant has in their store.
+
+`app\Http\Controllers\TenantProducts.php`
+```php
+public function index(Request $request)
+{
+  // We currently do not have a tenant identified as this is the "admin sub-site"
+
+  // Get our tenant (A Customer)
+  $customer = App\Models\Customer::firstOrFail($request->get('tenant_id'));
+  // Tell tenancy, that we want to access the above tenant
+  Tenancy::setTenant($customer)
+
+  // We are still in the "admin sub-site", but we now also have an active tenant
+  // And we can access the information stored in their database.
+
+  // We will assume that Products uses the `OnTenant` Trait
+  $tenant_products = Products::all();
+
+  // Return the products that belong to the tenant still within our "admin sub-site"
+  return $tenant_products
+
+}
+```
+
+## Considerations to take into account
+1. **Storing users in tenant databases**
+When the User Model has the `OnTenant` trait applied to it, you will not be able to have those users login to any "non-tenant" routes. Because there is no tenant identified, it will not be possible to authenticate the user.
+To overcome this you will need to either create two different User models (Admin & User), or will need to store all users in the system database. The option you choose will depend on youre goals and objectives.
```

Comment:
This is not true and reminds me a lot about the restrictions that hyn/multi-tenant had. If you would want an admin dashboard that allows authenticating users from a tenant, you can create a log in page that asks for their credentials and one more field with a slug, unique hash or something else to identify the tenant with. Then, modify the authentication layer to see whether the identifier is in the payload and set up a tenant database connection on the fly..

t/t is built to be as flexible as possible, the only limitation is your own imagination.